metadata-ingestion/src/datahub/ingestion/source/looker_common.py

```diff
@@ -507,34 +507,40 @@ def from_api(  # noqa: C901
                 model, explore_name, transport_options=transport_options
             )
             views = set()
-            if explore.joins is not None and explore.joins != []:
-                if explore.view_name is not None and explore.view_name != explore.name:
-                    # explore is renaming the view name, we will need to swap references to explore.name with explore.view_name
-                    aliased_explore = True
-                    views.add(explore.view_name)
-                else:
-                    aliased_explore = False
 
+            if explore.view_name is not None and explore.view_name != explore.name:
+                # explore is not named after a view and is instead using a from field, which is modeled as view_name.
+                aliased_explore = True
+                views.add(explore.view_name)
+            else:
+                # otherwise, the explore name is a view, so add it to the set.
+                aliased_explore = False
+                if explore.name is not None:
+                    views.add(explore.name)
```

Comment:
moved the extraction of the explore name (or aliased name) to the top level, outside of the as it always needs to be done regardless of whether there are joins.